maths-club-pack/content/student/minimax.md

```diff
@@ -0,0 +1,25 @@
+---
+title: MiniMax Algorithm
```

Comment:
the title will be converted into a slug and used to reference the markdown file, so `MiniMax Algorithm` => `minimax-algorithm` which doesn't match the filename. So please update to just be `Minimax` (you can still keep the display title in the markdown as Minimax Algorithm, this one is just for metadata extraction)